pom.xml

```diff
@@ -16,7 +16,10 @@
   <properties>
     <jenkins.version>1.609</jenkins.version>
     <java.level>6</java.level>
-    <workflow.version>1.4.3</workflow.version>
+    <workflow-job.version>1.4.3</workflow-job.version>
+    <workflow-cps.version>1.4.3</workflow-cps.version>
```

Comment:
`workflow-cps` version 2.10 works only with minimum Jenkins core version 1.642.1. This means that I need to bump the core version to 1.642.1 and java level to 7, which I don't think is a good idea, yet.